client/x/evmstaking/module/module.go

```diff
@@ -106,19 +103,14 @@ func (AppModule) IsOnePerModuleType() {}
 func (AppModule) IsAppModule() {}
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (AppModule) ConsensusVersion() uint64 { return ConsensusVersion }
+func (AppModule) ConsensusVersion() uint64 { return 1 }
```

Comment:
I think since we don't need to migrate any states, we can probably keep the consensus version the same. In this case, we don't need to add migrations.go file in the keeper

Reply:
18d87f5